nextcloud_update.sh

```diff
@@ -544,9 +544,20 @@ then
                 then
                     print_text_in_color "$IGreen" "Upgrading Bitwarden..."
                     sleep 2
-                    bash /root/bitwarden.sh updateself
-                    bash /root/bitwarden.sh update
+                    yes no | bash /root/bitwarden.sh updateself
+                    yes no | bash /root/bitwarden.sh update
                 fi
+            elif [ -d "$BITWARDEN_HOME"/bwdata ]
+                then
+                    curl_to_dir "https://raw.githubusercontent.com/bitwarden/server/master/scripts" "bitwarden.sh" "$BITWARDEN_HOME"
+                    chown "$BITWARDEN_USER":"$BITWARDEN_USER" "$BITWARDEN_HOME"
```

Comment:
I would only chown the downloaded script so:
`chown "$BITWARDEN_USER":"$BITWARDEN_USER" "$BITWARDEN_HOME"/bitwarden.sh`
or we could also chown the whole folder like this:
`chown -R "$BITWARDEN_USER":"$BITWARDEN_USER" "$BITWARDEN_HOME"`

Reply:
Additionally, I would make it exectuable with 
`chmod +x "$BITWARDEN_HOME"/bitwarden.sh`

Reply:
this is fixed